blocks/medium-promo-block/features/medium-promo/default.jsx

```diff
@@ -46,6 +92,10 @@ const MediumPromoItem = ({ customFields, arcSite }) => {
         type
         url
         lead_art {
+					_id
```

Comment:
Spacing looks off here, is this a tabs vs spaces difference?

Reply:
@matthewroach: Yes, thanks! Updated. The query was in spaces and my updates were tabbed. I updated so that the entire filter uses tabs for indention.